network/snat/snat_linux.go

```diff
@@ -46,6 +53,7 @@ type Client struct {
 	enableProxyArpOnBridge bool
 	netlink                netlink.NetlinkInterface
 	plClient               platform.ExecClient
+	IPTablesClient         ipTablesClientInterface
```

Comment:
Why is this in capital case ?

Reply:
The `IPTablesClient` variable is passed into `nuc.EnableIPForwarding` in endpoint_snatroute_linux.go (in the network package). Since network and snat are two different packages, in order to see the client, it needs to be exported (capitalized).

Reply:
Addressed by moving `EnableIPForwarding` to snatClient since that is the only place it is used and the function calls iptables and has logic specific to snat.